ee-feature-pack/common/src/main/resources/modules/system/layers/base/org/jboss/as/weld/jpa/main/module.xml

```diff
@@ -41,6 +41,7 @@
         <module name="org.jboss.as.jpa.spi"/>
         <module name="org.jboss.as.server"/>
         <module name="org.jboss.as.ee"/>
+	<module name="org.jboss.metadata.ear"/>
```

Comment:
Just a nitpick but indentation is off here.

Reply:
Thanks, updating to remove tabs and switch back spaces.